mypy.ini

```diff
@@ -0,0 +1,23 @@
+[mypy]
+python_version = 3.9
```

Comment:
Why 3.9? I know python 3.10 currently doesn't work with Modin. Will we have to update this?

Reply:
Reading the docs online, it looks like mypy will default to using whatever python version is used to run `mypy` itself. I set 3.9 because it seems it doesn't accept minor versions (e.g. 3.9.12) and I wasn't sure about 3.10. I could just get rid of this, but I figured that pinning a version is always better than not doing so.

Reply:
I would rather we don't pin the version, as pinned version prohibits from running the checks with whatever version it is installed.

As for 3.10 - I think Modin should support that (we should be pretty agnostic on upper bound, requiring >=3.8), it's Ray which doesn't support 3.10 now.